readthedocs/urls.py

```diff
@@ -20,7 +20,13 @@
 
 basic_urls = [
     url(r'^$', HomepageView.as_view(), name='homepage'),
-    url(r'^support/', TemplateView.as_view(template_name='support.html'), name='support'),
+    url(r'^support/success/$',
+        TemplateView.as_view(template_name='support/success.html'),
+        name='support_success'),
+    url(r'^support/error/$',
+        TemplateView.as_view(template_name='support/error.html'),
+        name='support_error'),
```

Comment:
How the users are redirected to these pages? Is this configured in front?

Reply:
Yep. I will note a comment.